internal/reflect/number_test.go

```diff
@@ -124,25 +103,6 @@ func TestNumber_int(t *testing.T) {
 	}
 }
 
-func TestNumber_intOverflow(t *testing.T) {
```

Comment:
Rather than removing tests, does it make sense to flip their expectations so we still have regression testing for these scenarios?

Reply:
Apologies if I've misunderstood, but by "flip their expectations" do you mean assert that errors are generated when there is an overflow? If so, my understanding was that tests such as `TestNumber_intOverflowError` were covering these cases.

Reply:
Ah okay, so these tests being removed were just covering the `AllowRoundingNumbers` usage to prevent errors which wasn't obvious from the prior test naming. Carry on!